docs/Tech_Grants_2021.md

```diff
@@ -189,5 +191,188 @@ Running:
 ```
 python nutrition_labels/predict.py --config_path configs/predict/2021.04.02.ini
 ```
-will predict tech grants for the dataset given in the `grants_data_path` config variable. This gave 2893 tech grants predicted in 16914 grants (17%) - these tagged grants are stored in `data/processed/predictions/210402/wellcome-grants-awarded-2005-2019_tagged.csv`.
+will predict tech grants for the dataset given in the `grants_data_path` config variable.
+
+#### Evaluation on model trained with 360Giving data
+
+```
+python nutrition_labels/evaluate.py --config_path configs/evaluation/2021.04.04.ini
+```
+When training on 360Giving data, the evaluation results on the test dataset are exactly the same as above. The results for evaluation on RF/not used were also the same, but the EPMC evaluation dropped to 58%.
+
+### Results
+
+Results discussed here will be for both the `configs/predict/2021.04.06.ini` model predictions - which is trained and predicted on the 360Giving open dataset, and the `configs/predict/2021.04.03.ini` model predictions - which is trained and predicted on all the grant data from FortyTwo, as downloaded on 20th April 2021.
+
+The results for these two models/predictions are very similar - so we will comment on both, but give additional information (e.g. plots) for the 360Giving version since this is reproducible externally too.
+
+#### Number of tech grants predicted
+
+The `configs/predict/2021.04.06.ini` config file contains the path to the 360Giving dataset of 16,914 grants. The model predicted that 21% (3572) of these are tech grants - these tagged grants are stored in `data/processed/predictions/210406/wellcome-grants-awarded-2005-2019_tagged.csv`. This took about 3 hours to predict on my machine.
+
+The `configs/predict/2021.04.03.ini` config file contains the path to all the grant data from fortytwo, as downloaded on 20th April 2021, this contains 126,341 grants. The model predicted that 12% (15199) of these are tech grants - these tagged grants are stored in `data/processed/predictions/210403/all_grants_fortytwo_info_210420_tagged.csv`. This took about 12 hours to predict on my machine.
```

Comment:
Yeh, I thought it might be because the 360 giving is just from 2005-2019, it misses out those historic grants which might be more likely to not produce tech.

I just did a little check - 16853 grants are in both the 360giving and 42 data. These agree on their predicted classification 94% of the time. So yeh, seems to be something about the differences in the additional grants in the 42 data, it'd be a good thing to look into properly.